lib/plugins/aws/remove/lib/ecr.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = async function () {
+  // TODO: TESTS
+  // TODO: ADD LOGGING
+  // TODO: CHECK IF THERE ARE ANY IMAGES IN SERVERLESS.YML?
+  const registryId = await this.provider.getAccountId();
+  const repositoryName = this.provider.getEcrRepositoryName();
+  const params = {
+    registryId,
+    repositoryName,
+    force: true, // To force removal of non-empty repository
+  };
+
+  try {
+    await this.provider.request('ECR', 'deleteRepository', params);
+  } catch (err) {
+    if (err.providerError && err.providerError.code === 'RepositoryNotFoundException') {
+      // Pass silently
+    } else {
+      throw err;
```

Comment:
I wonder whether it's fine to unconditionally `throw` here.

e.g. what if user works in region in which AWS ECR is not supported at all (I'm not sure if it can be the case, but if it would we will throw at every removal in such region.

Reply:
I did not consider that situation, I'll investigate if such a thing can happen

Reply:
I tried to investigate it but I didn't find a single mention of ECR not being available in a specific region - do you envision any other potential issues with unconditionally throwing here @medikoo ?

Reply:
I saw you already updated to issue it only if it's confirmed that repository exists. In such case I think it's totally safe to run it unconditionally

Reply:
That's correct, however, checking for repository existence has similar check at the moment, which can trigger (potentially) the same issue

Reply:
Yes, I just saw that. I think we can leave it for now, and eventually we'll react if anything is reported.

I'm not sure how it'll behave in e.g. Chinese regions, which are not accessible if you do not have registered business in China

Reply:
I was thinking about China specifically, but if you don't have access there, how all other API calls via SDK work?

Reply:
I was referring more to case where someone has access but e.g. given AWS service is not provided in the region (I can imagine that china regions could be limited to US ones).

Reply:
Understood :+1: I was actually researching availability of ECR service but didn't find a single region where it's not available, including China regions: https://www.amazonaws.cn/en/about-aws/regional-product-services/